devimint/src/main.rs

```diff
@@ -175,16 +178,29 @@ pub async fn latency_tests(dev_fed: DevFed) -> Result<()> {
     let reissue_stats = stats_for(reissues);
     let ln_sends_stats = stats_for(ln_sends);
     let ln_receives_stats = stats_for(ln_receives);
+
+    info!("Testing latency of restore");
+    let backup_secret = cmd!(backup_client, "print-secret").out_json().await?["secret"]
+        .as_str()
+        .map(ToOwned::to_owned)
+        .unwrap();
+    let _ = cmd!(backup_client, "wipe", "--force",).run().await?;
+    let start_time = Instant::now();
+    let _ = cmd!(backup_client, "restore", &backup_secret).run().await?;
+    let restore_time = start_time.elapsed();
+
     println!(
         "================= RESULTS ==================\n\
               REISSUE: {reissue_stats}\n\
               LN SEND: {ln_sends_stats}\n\
-              LN RECV: {ln_receives_stats}"
+              LN RECV: {ln_receives_stats}\n\
+              RESTORE: {restore_time:?}"
     );
     // FIXME: should be smaller
     assert!(reissue_stats.median < Duration::from_secs(4));
     assert!(ln_sends_stats.median < Duration::from_secs(6));
     assert!(ln_receives_stats.median < Duration::from_secs(6));
+    assert!(restore_time < Duration::from_secs(160));
```

Comment:
It was 83 on my machine, I multiplied with factor of 2 to account for CI slow down.

Reply:
This may or may not work as you could have been at an arbitrary point inside the current session. If you start restoring at the end of a session it will take shorter, if you start at the beginning it will take longer. https://github.com/fedimint/fedimint/pull/3956#issuecomment-1860148765

But let's just see how it does on master and narrow down the allowed time as we optimize restoring.